src/cpp/desktop/DesktopWebPage.cpp

```diff
@@ -466,7 +443,13 @@ bool WebPage::acceptNavigationRequest(const QUrl &url,
       {
          return true;
       }
-      else
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
+      else if (navType == NavigationTypeRedirect)
+      {
+         return true;
+      }
+#endif
+      else if (navType == NavigationTypeLinkClicked)
```

Comment:
Nervous about relying on navType in Qt 5.12.8, just because it introduces the need to retest this on all platforms. How about this just stays as "else", thus wrapping all use of navType in the QT_VERSION >= 5.15 and making this change a no-op for anything but Mac w/5.15?